### DISCUSSION TITLE
Q: 用Docker部署的Waline如何迁移？ || Q: How to migrate Waline deployed with Docker?

### DISCUSSION THREAD

**Discussion by Simonzhang8** [walinejs/waline#2728] [Q&A] [answered]
### Describe the bug | 问题描述

原本是用Docker+MySQL部署的Waline，现在服务器到期更换了另一家的服务器，希望在新服务器上继续用Docker+MySQL的方法部署Waline，需要怎么操作？

### Website URL | 问题网站

略

### Where your waline deploy? | 服务部署在哪里？

Docker

### Where your comment data store? | 数据存储在哪里？

MySQL

---
### Describe the bug | Problem description

Waline was originally deployed using Docker+MySQL. Now the server has expired and has been replaced by another server. I hope to continue to use Docker+MySQL to deploy Waline on the new server. What should I do?

### Website URL | Problem website

slightly

### Where your waline deploy? | Where is the service deployed?

Docker

### Where your comment data store? | Where is the data stored?

MySQL

**Comment by lizheming** [comment, marked as answer]
直接把你的 MySQL 存储的本地文件迁移过去就可以了吧，Waline 本身没有什么额外的存储

---
Just migrate your local files stored in MySQL there. Waline itself does not have any additional storage

**Reply by Simonzhang8** [reply]
> 直接把你的 MySQL 存储的本地文件迁移过去就可以了吧，Waline 本身没有什么额外的存储
> 
> Just migrate your local files stored in MySQL there. Waline itself does not have any additional storage

好的。我想知道除了数据库文件迁移过去，Waline程序上还需要有哪些操作？还是说填写相同的数据库信息后就直接可以继续使用了？

**Reply by lizheming** [reply]
没有了

---
No more

**Reply by Simonzhang8** [reply]
> 没有了
> 
> No more

好的，非常感谢🙏

---
> No more
>
> No more

OK, thank you very much🙏
